toolkits/springernature/packages/springernature-submission-header/scss/50-components/core.scss

```diff
@@ -0,0 +1,24 @@
+.c-submission-details-header {
+	font-size: 12px;
+	font-size: 0.75em;
+	margin: 0 0 20px 0;
+	background: #fff;
+	padding: 20px;
+	border: 1px solid #173643;
```

Comment:
do you have border colours defined in your brand context? as they could be referenced here

Reply:
This colour is not included in the brand context colours